### ISSUE TITLE
Large videos cause the playback to stutter

### ISSUE THREAD

**Issue by flocked** [vitoziv/VIMediaCache#70]
Caching a video like this

> https://sylvan.apple.com/Aerials/2x/Videos/LA_A008_C004_4K_SDR_HEVC.mov

causes the playback to stutter. I think it happens because AVPlayer can't catch up with writing of the big cache file.

**Comment by lexrus** [comment]
Yep, it has to create a 793MB dummy cache file before playback...
